scripts/code_review_tool_evaluator.py

```diff
@@ -72,7 +72,7 @@ def get_file(commit_hash, path):
             (
                 "RAG",
                 code_review.CodeReviewTool(
-                    llm=llm,
+                    comment_gen_llms=[llm],
```

Comment:
Why do we pass the llm as a list here?

Reply:
The idea would be to support an ensemble of LLMs (either the same model with different temperature and top_p, or different models) to generate diverse comments, and then deduplicate the comments.

Reply:
We may have such implication details in a class that uses the LLM interface. So this class doesn't need to know how the comments were generated.

Reply:
Good idea, filed #4539 as a follow-up to do this.